Issue3 introduction general (#8)
metno/data-management-handbook: doc/introduction.adoc

[[introduction]]
== Introduction
:xrefstyle: short

//not updated: chapter links, check all links to glossary and acronyms
//remember: change references to quick manual

The purpose of the Data Management Handbook (DMH) is threefold:

1. to provide an overview of the principles for data management to be employed.
2. to help personnel identify their roles and responsibilities for good data management.
3. to provide personnel with practical guidelines for carrying out good data management.

_Data management_ is the term used to describe the handling of data in a systematic and cost-effective manner. 
The data management regime should be continuously evolving, to reflect the evolving nature of data colletion. Therefore this DMH is a living document that will be revised and updated from time to time in order to maintain its relevance.

The DMH is a strategic governing document and should be used as part of the quality framework the organisation is using. 

// Remember to add links/references to the chapters below

> This DMH focuses on the management of geophysical data, types of data and the usage of these data are described in the organisation specific information in Section X.X.X.

Reviewer: I think something is odd with this sentence. Maybe split it in two?

```suggestion
This DMH focuses on the management of geophysical data. Types of data and the usage of these data are described in the organisation specific information in Section X.X.X.
```

This introduction (Chapter 1) lays forth the background and principles for the data management regime. 
Chapters 2-5 describe the implementation of the main building blocks: structuring and documenting data (Chapter 2), data services (Chapter 3), portals and documentation aimed at users (Chapter 4) and governance issues (Chapter 5). 
Each chapter starts with a brief statement of its purpose, followed by a description of what is implemented at the organisation at present, as well as the planned developments for the short-term (<2 years) and expected developments for the longer term (2-5 years). 
Practical guidelines for carrying out good data management are addressed in Chapter 6 and especially in the Quick Manual for Data Providers. 
The Quick Manual is a concise, informal HOW-TO for data management practitioners.

//Remember to change if we move away from the Quick manual

The intended audience for this DMH is any co-worker who is involved in any part of the value chain for geophysical data (<<img-value_chain>>).

The handbook can be used in three ways: 

1. Read the Introduction (Chapter 1) to find out the background and principles of data management;

> 2. Read Chapters 2-5 to learn about how data management is currently implemented and how it is expected to evolve in the next few years:

Reviewer: ```suggestion
2. Read Chapters 2-5 to learn about how data management is currently implemented and how it is expected to evolve in the next few years;
```

> 3. Go to the Quick Manual for Data Providers for guidelines on what to do in real-life cases; alternatively Chapter 6 for typical workflow examples.

Reviewer: ```suggestion
3. Read the Quick Manual for Data Providers for guidelines on what to do in real-life cases; alternatively read Chapter 6 for typical workflow examples.
```

=== The principles of data management for geophysical data

One of the main motivations for implementing a unified data management model is to better serve the users of the data. Primarily, this can be approached by making user needs and requirements the guide for determining what data we provide and how. For example, it will be described below how the specification of datasets should be determined by user needs. By implementing the data management practices described here, it is expected that users will benefit from:

* the ease of discovering, viewing and accessing all the datasets that are offered by the institute;
* standardised ways of downloading data, which reduces the need for special solutions on the user side;
* reducing their own data storage needs, by downloading just what they need;
* easy and standardised access to remote datasets and catalogues, when using their own visualisation/analysis tools;
* the ability to compare and combine MET Norway’s data with data from other sources (through metadata catalogues);
* the ability to apply common data transformations, like spatial, temporal and variables subsetting and reprojection, before downloading anything;

> *the ease of building specific metadata catalogues and data portals that include data from MET Norway and can target a specific user community;

Reviewer: ```suggestion
* the ease of building specific metadata catalogues and data portals that include data from MET Norway and can target a specific user community;
```

> *the ease of building specific metadata catalogues and data portals that include data from MET Norway and can target a specific user community;

Reviewer: ```suggestion
*the ease of building specific metadata catalogues and data portals that include data from the institute and can target a specific user community;
```

//link to proper section

[[geophysical-value-chain]]
==== The geophysical value chain

An example of a geophysical value chain is presented in Figure 1. Typically, data from a wide variety of providers are used in the value chain. Traditionally, the data used have been transmitted on request from one data centre to another, and used in the specific processing chains that requested the data. 
The focus on reuse of data in various contexts has been missing.

// need to solve the link to the image and possibly also the linking to the image

[#img-value_chain]
.Value chain for geophysical data
image::value_chain.png[Value chain]

At the end of the data management value chain are the users of the data (aka. data consumers, see Section 1.2.2.1), who may be either external or internal to the institute.

[[fair-data-management-model]]
==== A data management model based on the FAIR principles

This model is based on the model of the Arctic Data Centre, which adheres to the FAIR principles. The model’s basic functions fall into three main categories:

1. *Documentation of data* using discovery and use metadata (metadata are further described below). 

> The documentation identifies who, what, when, where, and how and shall make it easy for consumers to find and understand data. This requires application of information containers and utilisation of controlled vocabularies and ontologies where textual representation is required. It also covers the topic of data provenance which is used to describe the origin and all actions done on a dataset. Data provenance is closely linked with workflow management. Furthermore, it covers the relationship between datasets. Application of ontologies in data documentation is closely linked to the concept of linked data. 

Reviewer: ```suggestion
The documentation identifies who, what, when, where, and how, and shall make it easy for consumers to find and understand data. This requires application of information containers and utilisation of controlled vocabularies and ontologies where textual representation is required. It also covers the topic of data provenance which is used to describe the origin and all actions done on a dataset. Data provenance is closely linked with workflow management. Furthermore, it covers the relationship between datasets. Application of ontologies in data documentation is closely linked to the concept of linked data. 
```

.Brief introduction to different types of metadata.
[[tabl-metadata]]
[%header, cols=4*]
|===
|Type
|Purpose
|Description
|Examples

|[[discovery-metadata]]Discovery metadata
|Used to find relevant data
|Discovery metadata are also called index metadata and are a digital version of the library index card. They describe who did what, where and when, how to access data and potential constraints on the data. They shall also link to further information on the data like site metadata. Discovery metadata are thus WIS metadata.
|ISO 19115
GCMD DIF

|[[use-metadata]]Use metadata
|Used to understand data found
|Use metadata describe the actual content of a dataset and how it is encoded. The purpose is to enable the user to understand the data without any further communication. They describe the content of variables using standardised vocabularies, units of variable, encoding of missing values, map projections, etc.
|Climate and Forecast (CF) Convention
BUFR
GRIB

|[[configuration-metadata]]Configuration metadata

> |Used to tune portal services for datasets for users.

Reviewer: ```suggestion
|Used to tune portal services for datasets for users
```

|Configuration metadata are used to improve the services offered through a portal to the user community. This can be e.g. how to best visualise a product.
|
 

> |[[site-metadata]]Site metadata

Reviewer: I would put site metadata just after use metadata as they are kind of related. Then put the configuration metadata and system metadata.

Author: I will move this in the next issue

|Used to understand data found
|Site metadata are used to describe the context of observational data. They describe the location of an observation, the instrumentation, procedures, etc. To a certain extent they overlap with discovery metadata, but also extend the discovery metadata. Site metadata can be used for observation network design. Site metadata can be considered a type of use metadata.
|WIGOS
OGC O&M
StInfoSys

|[[system-metadata]]System metadata
|Used to understand the technical structure of the data management system and track changes in it 

> |System metadata covers e.g. technical details of the storage system (e.g. Lustre metadata), web services their purpose and how they interact with other components of the data management system, available and consumed storage, number of users and other KPI elements etc.

Reviewer: ```suggestion
|System metadata covers e.g. technical details of the storage system (e.g. Lustre metadata), web services, their purpose and how they interact with other components of the data management system, available and consumed storage, number of users and other KPI elements etc.
```
